conf/DEV.conf

```diff
@@ -13,4 +13,6 @@ subscriptions.url="https://sub.thegulocal.com"
 
 play.ws.acceptAnyCertificate=true
 
+touchpoint.backend.default=DEV
+
 sentry.dns = ""
```

Comment:
Just spotted a typo (pre-dating this PR): this should be "sentry.dsn" not sentry.dns!